Makefile

```diff
@@ -83,6 +85,9 @@ clean:
 
 binaries: cri-containerd
 
+static-binaries: GO_LDFLAGS=--ldflags '-extldflags "-fno-PIC -static"'
```

Comment:
didn't realize where you were heading with this.. can we make these LD flags more specific and leave the environment variables to other uses?  say make your flags GO_STATIC_LDFLAGS?